backends/occa/ceed-occa-restrict.okl

```diff
@@ -87,3 +87,20 @@
     }
   }
 }
+// *****************************************************************************
+@kernel void kRestrict5b(const int ncomp,
```

Comment:
@camierjs if you get a moment, can you look at this? I just wrote the first test for a restriction with interlaced elements and OCCA's kRestrict5b (I just added this) and/or kRestrict2 to handle restrictions with lmode==CEED_TRANSPOSE aren't behaving correctly.

Reply:
I got it